modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/ChangeTrackingIndicator.js

```diff
@@ -454,6 +485,48 @@ export default function ChangeTrackingIndicator({
 						</ClayLayout.Col>
 					</ClayLayout.Row>
 
+					{contextChangeButtons && (
+						<>
+							<ClayLayout.Row
+								style={{marginBottom: '8px', marginTop: '16px'}}
+							>
+								<ClayLayout.Col
+									style={{
+										alignItems: 'center',
+										display: 'flex',
+									}}
+								>
+									<ClayCheckbox
+										checked={popoverCheckbox}
+										label={Liferay.Language.get(
+											'do-not-show-this-message-again-in-the-selected-period-of-time'
+										)}
+										onChange={() =>
+											setPopoverCheckbox(!popoverCheckbox)
+										}
+										style={{marginLeft: '10px'}}
+									/>
+
+									<ClaySelectWithOption
+										id="silenceContextChangePopover"
+										onChange={handleSelectChange}
```

Comment:
Since `handleSelectChange` just calls `setSilenceContextChangePopover`, we can inline it without having an additional function. You can get the event value like this

```
onChange={(event) => {
	setSilenceContextChangePopover(event.target.value);
}}
```